models/spring-ai-minimax/src/test/java/org/springframework/ai/minimax/api/MiniMaxApiToolFunctionCallIT.java

```diff
@@ -126,7 +127,7 @@ public void toolFunctionCall() {
 
 		assertThat(chatCompletion2.getBody().choices().get(0).message().role()).isEqualTo(Role.ASSISTANT);
 		assertThat(chatCompletion2.getBody().choices().get(0).message().content()).contains("San Francisco")
-			.containsAnyOf("30.0°C", "30°C", "30.0°F", "30°F");
+			.containsAnyOf("30.0°C", "30°C");
```

Comment:
I had to update to use 

```
				.containsAnyOf("30.0°C", "30°C", "30.0")
				.containsAnyOf("°C", "Celsius");
```
as the answer said "30 Celcius"

also, it seems sometimes i get an answer back in chinese.  Is there a way for it to be force to reply in english?

Reply:
as far as i know, there's can't to set the talk language through options, but we can add an explicit request in the message, like `please answer in english`